app/views/roasts/index.html.erb

```diff
@@ -0,0 +1,49 @@
+<%= render "shared/partials/nav-global" %>
+<%= render "shared/partials/header-global" %>
+
+<!-- first loop around main and second loop around roasts_brands section -->
+<!-- can loop the markup through Rails since each roast is templated -->
+
+<section class="roasts__slider-wrapper">
+    <div class="slider-axis">
+      <%= image_tag("light-slider-bean.svg", alt:"") %>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <span></span>
+      <%= image_tag("dark-slider-bean.svg", alt:"") %>
+    </div>
+    <div class="slider"><input type="range" value="0" min="0" max="8" step="1" class="slider__range"></div>
+</section>
+
+<main class = "roasts">
```

Comment:
`<main>` should be around the entire page that isn't the global header and footer.

Also - get rid of those spaces on your class attribute: `class="roasts"`